src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -586,6 +591,8 @@ Putting It All Together
 
       You will notice that there is a dependency which looks like this:
 
+      .. TODO Check that this happens with the current Cylc Graph
```

Comment:
TODO

Reply:
I don't think that it does - do you want me to change to documentation or raise an issue to changethe behaviour of the interim Cylc Graph

Reply:
If there's a bug with cylc graph raise it on cylc-flow (we aren't going to spot a TODO in the docs for a while).

Reply:
https://github.com/cylc/cylc-flow/issues/4638